ivy/data_classes/container/experimental/losses.py

```diff
@@ -0,0 +1,162 @@
+# global
+from typing import Optional, Union, List, Dict
+
+# local
+import ivy
+from ivy.data_classes.container.base import ContainerBase
+
+
+class _ContainerWithLosses(ContainerBase):
+    @staticmethod
+    def _static_huber_loss(
+        true: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        pred: Union[ivy.Container, ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        delta: Union[float, ivy.Container] = 1.0,
```

Comment:
```suggestion
        delta: Union[float] = 1.0,
```

Only arrays arguments can be containers.